x/curating/abci_test.go

```diff
@@ -85,22 +88,26 @@ func TestEndBlockerExpiringPost(t *testing.T) {
 
 	curating.EndBlocker(ctx, app.CuratingKeeper)
 
-	// creator match reward = 0.5 * match_reward = 3 STB
-	creatorBal := app.BankKeeper.GetBalance(ctx, addrs[0], "ustb")
-	require.Equal(t, "10300000", creatorBal.Amount.String(),
-		"10 (initial) + 0.3 (creator match reward)")
+	creatorBal := app.BankKeeper.GetAllBalances(ctx, addrs[0])
+	// creatord voting reward = 0.3% * voting pool
+	require.Equal(t, "10030000", creatorBal.AmountOf("uatom").String(),
+		"10 (bal) + 30000uatom (creator voting reward)")
+
+	// creator protocol reward = 0.5 * match_reward = 3 STB
+	require.Equal(t, "10300000", creatorBal.AmountOf("ustb").String(),
+		"10 (bal) + 0.3 (creator protocol reward)")
 
 	curator1Bal := app.BankKeeper.GetAllBalances(ctx, addrs[1])
-	require.Equal(t, "14000000", curator1Bal.AmountOf("uatom").String(),
-		"9 (bal) + 5 (voting reward)")
-	require.Equal(t, "12850000", curator1Bal.AmountOf("ustb").String(),
-		"9 (bal) + 1 (deposit) + 2.85 (match reward)")
+	require.Equal(t, "13985000", curator1Bal.AmountOf("uatom").String(),
+		"9 (bal) + ? (voting reward)")
```

Comment:
Did you mean to have a `?` hear or was that supposed to be a placeholder?

Reply:
Ah good catch. Will fill in.